extraction/formulate.api.md

```diff
@@ -0,0 +1,29 @@
+## API Report File for "@department-of-veterans-affairs/formulate"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
```

Comment:
Oh interesting. Probably not part of `lint-staged` since that just works on the changed hunks, but possibly as a pre-commit hook. Possibly as a GH action on `push` events. Possibly as a GH action only after merging the branch. :thinking: It'll take some thinking and a decision document, I'd say. (Which is great fodder for the `docs/explanations/` :grin: )

Reply:
How do you feel about renaming the directory this gets put in? `extraction` is a bit vague, I think. :thinking: What might be clearer to somebody checking out the repo with no context?

Reply:
I'm fine with renaming it. Would `api-docs` or `api-extraction` be better?